packages/Webkul/Admin/src/Resources/views/catalog/products/edit/types/configurable.blade.php

```diff
@@ -817,6 +817,14 @@ class="inline-block text-violet-700 hover:text-violet-700 hover:underline"
                     }
 
                     for (const attribute of this.superAttributes) {
+                        if( params[attribute.code].length === 0){
+                            setErrors({
+                                [attribute.code]:"@lang('admin::app.catalog.products.edit.types.configurable.variant-attribute-option-not-selected')",
```

Comment:
It would be better to use the required validation message here instead of custom as this is only for blank field.

Reply:
added required validation